jupyterlab/stage3-py.sh

```diff
@@ -0,0 +1,92 @@
+#!/bin/sh
+set -e
+#This commented-out bit, plus changing the definition of LOADRSPSTACK in
+# Dockerfile.template, will clone the environment rather than installing
+# into the stack environment itself.  This adds 50% or so to the container
+# size.
+#
+# source ${LOADSTACK}
+# rspname="rsp-${LSST_CONDA_ENV_NAME}"
+# conda create --name ${rspname} --clone ${LSST_CONDA_ENV_NAME}
+#
+source ${LOADRSPSTACK}
+conda install -y mamba # not strictly necessary, but better error reporting
+mamba install -y \
+      'jupyterlab=2' \
+      jupyterhub \
+      jupyter-server-proxy \
+      jupyter-packaging \
+      geoviews \
+      cookiecutter \
+      nbval \
+      pyshp \
+      pypandoc \
+      astroquery \
+      ipyevents \
+      ipywidgets \
+      ipyevents \
+      bokeh \
+      cloudpickle \
+      ipympl \
+      fastparquet \
+      paramnb \
+      ginga \
+      bqplot \
+      ipyvolume \
+      papermill \
+      'dask=2020.12' \
+      gcsfs \
+      snappy \
+      'distributed=2020.12' \
+      dask-kubernetes \
+      "holoviews[recommended]" \
+      datashader \
+      python-snappy \
+      graphviz \
+      'mysqlclient!=2.0.2' \
+      hvplot \
+      intake \
+      intake-parquet \
+      jupyter-server-proxy \
+      toolz \
+      partd \
+      nbdime \
+      dask_labextension \
+      numba \
+      awkward \
+      awkward-numba \
+      swifter \
+      pyvo \
+      'jupyterlab_iframe<0.3' \
+      astrowidgets \
+      sidecar \
+      python-socketio \
+      pywwt \
+      freetype-py \
+      terminado \
+      nodejs \
+      yarn \
+      "jedi<0.18.0"
+# These are the things that are not in conda.
+pip install --upgrade \
+       lsst-efd-client \
+       wfdispatcher \
+       firefly-client \
+       socketIO-client \
+       rubin_jupyter_utils.lab \
+       jupyterlab_hdf \
+       jupyter_firefly_extensions \
+       nbconvert[webpdf] \
+       nclib \
+       git+https://github.com/ericmandel/pyjs9
+
+# Add stack kernel
+python3 -m ipykernel install --name 'LSST'
+
+# Remove "system" kernel
```

Comment:
It did not--but everything in the system python environment has to be installed via pip (because no conda).  When we were also installing into the stack python with pip, it was pretty much "install into system; activate stack; do the same thing."  If we kept the system python we'd be doing two very different things in those environments now.  Instead, my approach is to have the ability to start a "system" python as a cloned conda env and install different things into it (as described above).  At the moment, we have no need for that.